tests/flow/test_graph_info.py

```
@@ -0,0 +1,436 @@
+                  "Execution duration", "Report duration", "Utilized cache",
+                  "Write", "Timeout"]
+        assert(all(field in event for field in fields))
+
+        # cast and initialize
+        self.received_at        = datetime.datetime.fromtimestamp(int(event['Received at']))
+        self.query              = event['Query']
+        self.total_duration     = float(event['Total duration'])
+        self.wait_duration      = float(event['Wait duration'])
+        self.execution_duration = float(event['Execution duration'])
+        self.report_duration    = float(event['Report duration'])
+        self.utilized_cache     = False if event['Utilized cache'] == '0' else True
```

Ruff (0.12.2)

Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)